suite-native/module-settings/src/screens/SettingsPrivacyAndSecurity.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { ReactNode } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { Screen, ScreenHeader } from '@suite-native/navigation';
+import { disableAnalyticsThunk, enableAnalyticsThunk } from '@suite-native/analytics';
+import { selectIsAnalyticsEnabled } from '@suite-common/analytics';
+import { Box, Card, DiscreetCanvas, Text, useDiscreetMode } from '@suite-native/atoms';
+import { useNativeStyles } from '@trezor/styles';
+
+import { TouchableSwitchRow } from '../components/TouchableSwitchRow';
+
+const RowDescription = ({ children }: { children: ReactNode }) => (
+    <Text variant="hint" color="textSubdued">
+        {children}
+    </Text>
+);
+
+const DiscreetTextExample = () => {
+    const { utils } = useNativeStyles();
+
+    return (
+        <DiscreetCanvas
+            text="$100"
+            color="textSubdued"
+            width={30}
+            fontSize={utils.typography.hint.fontSize}
+            height={utils.typography.hint.lineHeight}
+        />
+    );
+};
+
+export const SettingsPrivacyAndSecurity = () => {
+    const dispatch = useDispatch();
+    const isAnalyticsEnabled = useSelector(selectIsAnalyticsEnabled);
+    const { isDiscreetMode, setIsDiscreetMode } = useDiscreetMode();
+
+    const handleAnalyticsChange = (isEnabled: boolean) => {
+        if (isEnabled) {
+            dispatch(enableAnalyticsThunk());
+            return;
+        }
+        dispatch(disableAnalyticsThunk());
+    };
+
+    return (
+        <Screen header={<ScreenHeader title="Privacy & Security" />}>
+            <Card>
+                <TouchableSwitchRow
```

Comment:
Could you make small components of all these Rows to avoid rerenders of whole screen when anything is switched?

Keep them is this file, no need to separate them.

Reply:
Yeah, good idea
https://github.com/trezor/trezor-suite/pull/7883/commits/26d57a5343d89293463b6fe2ca52b7b7325e06ee